### DISCUSSION TITLE
Add the path $HOME/.local/share/icons for Qtile to look for icons

### DISCUSSION THREAD

**Discussion by cr1ogen** [qtile/qtile#3307] [Ideas] [answered]
As a result of this conversation https://www.reddit.com/r/qtile/comments/skpb20/widgetcurrentlayouticon_doesnt_work_with_custom/ , I wanted to propose if you can add the path so that Qtile also looks for the icons in `$HOME/.local/share/icons`  for Qtile to look for icons, since many programs look for the icons there. I wanted to make a PR but I didn't understand how it was done, so I propose it here. Thanks

**Comment by elParaguayo** [comment, marked as answer]
https://github.com/qtile/qtile/pull/3309

**Reply by m-col** [reply]
Merged now!

**Reply by cr1ogen** [reply]
Thanks
